Library/Homebrew/cmd/update-report.rb

```diff
@@ -154,7 +154,8 @@ def update_report
             puts_stdout_or_stderr
             puts_stdout_or_stderr <<~EOS
               You have #{msg} installed.
-              You can update #{update_pronoun} with #{Tty.bold}brew upgrade#{Tty.reset}.
+              You can upgrade #{update_pronoun} with #{Tty.bold}brew upgrade#{Tty.reset}
+              or check #{update_pronoun} with #{Tty.bold}brew outdated#{Tty.reset}.
```

Comment:
How about "list" instead of "check"? I'm not really sure anything is really being "checked" by running `brew outdated`.

```suggestion
              or list #{update_pronoun} with #{Tty.bold}brew outdated#{Tty.reset}.
```